feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -460,6 +463,217 @@ func configACLInterface(t *testing.T, iFace *oc.Acl_Interface, ifName string) *a
+			map[string]any{
+				"action": map[string]any{
+					"drop": map[string]any{},
+				},
+				"match": map[string]any{
+					"destination-ip": map[string]any{
+						"prefix": ateDstCIDR,
+					},
+					"source-ip": map[string]any{
+						"prefix": aclNullPrefix,
+					},
+				},
+			},
+		}
+		entry10Update, err := json.Marshal(aclEntry10Val)
+		if err != nil {
+			t.Fatalf("Error with json Marshal: %v", err)
+		}
+
+		var aclEntry20Val = []any{
+			map[string]any{
+				"action": map[string]any{
+					"drop": map[string]any{},
+				},
+				"match": map[string]any{
+					"source-ip": map[string]any{
+						"prefix": ateDstCIDR,
+					},
+					"destination-ip": map[string]any{
+						"prefix": aclNullPrefix,
+					},
+				},
+			},
+		}
+		entry20Update, err := json.Marshal(aclEntry20Val)
+		if err != nil {
+			t.Fatalf("Error with json Marshal: %v", err)
+		}
+
+		var aclEntry30Val = []any{
+			map[string]any{
+				"action": map[string]any{
+					"accept": map[string]any{},
+				},
+				"match": map[string]any{
+					"source-ip": map[string]any{
+						"prefix": aclNullPrefix,
+					},
+					"destination-ip": map[string]any{
+						"prefix": aclNullPrefix,
+					},
+				},
+			},
+		}
+		entry30Update, err := json.Marshal(aclEntry30Val)
+		if err != nil {
+			t.Fatalf("Error with json Marshal: %v", err)
+		}
+
+		gpbSetRequest := &gpb.SetRequest{
+			Prefix: &gpb.Path{
+				Origin: "srl",
+			},
+			Update: []*gpb.Update{
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "10"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry10Update,
+						},
+					},
+				},
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "20"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry20Update,
+						},
+					},
+				},
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "30"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry30Update,
+						},
+					},
+				},
+			},
+		}
+		gnmiClient := d.RawAPIs().GNMI().Default(t)
+		if _, err := gnmiClient.Set(context.Background(), gpbSetRequest); err != nil {
+			t.Fatalf("Unexpected error configuring SRL ACL: %v", err)
+		}
+	default:
+		t.Fatalf("Unsupported vendor, got vendor: %s, want: NOKIA", d.Vendor())
+	}
+}
+
+// Helper function to replicate AdminAllACL() configs in native model
+// Remove ACL entries 10, 20
```

Comment:
should it be this? "Helper function to replicate AdminAllACL() configs in native model, but remove ACL entries 10, 20"

Reply:
Yes it should, will fix typo!